### PR TITLE
fix(plugin-meeting): media ready event not triggring

### PR DESCRIPTION
**Pull request by arun3528**
Fixes #https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-170709

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 1]
```diff
@@ -1604,6 +1604,7 @@ export default class Meeting extends StatelessWebexPlugin {
     }
     // eslint-disable-next-line no-param-reassign
     pc.ontrack = (event) => {
+      console.log(event);
```
Comment (akoushke) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js]: shouldnt this be removed?

**Review by holsted — APPROVED** [review]

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 2]
```diff
@@ -1616,9 +1616,35 @@ export default class Meeting extends StatelessWebexPlugin {
...
+        }
+        else
+        if (mediaTrack.id === videoTransceiver.receiver.track.id) {
+          trackMediaID = '1';
+        }
+        else
+        if (mediaTrack.id === shareTransceiver.receiver.track.id) {
+          trackMediaID = '2';
+        }
+        else {
```
Comment (akoushke) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:1644]: do we still need this?

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 3]
```diff
@@ -1616,9 +1616,35 @@ export default class Meeting extends StatelessWebexPlugin {
         VIDEO_TRACK: '1',
         SHARE_TRACK: '2'
       };
-      const trackMediaID = event.transceiver.mid;
       let eventType = null;
       const mediaTrack = event.track;
+      let trackMediaID = null;
+
+
+      // **In case of safari some time the transceiver is not present for specific os version
```
Comment (akoushke) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:1624]: can you remove this `**` please? 🙈

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js (hunk 2 above)
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:1644]: just a precaution , dont want it to break with no reason

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js (hunk 3 above)
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:1624]: we need this  to track this is a hack for safari

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js (hunk 1 above)
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js]: removed

**Review by mwegman — APPROVED** [review]

**Pull request merged**